### ISSUE TITLE
[BUG] Tutorial creation errors with MULTISESSION_MODE = 2

### ISSUE THREAD

**Issue by kcoram** [evennia/evennia#2252]
#### Describe the bug
Running 'batchcommand tutorial_world.build' in a new game world configured with MULTISESSION_MODE = 2 generates an unexpected stack trace:

```
Traceback (most recent call last):
  File "/home/kevin/Source/muddev/evennia/evennia/commands/cmdhandler.py", line 640, in _run_command
    yield cmd.parse()
  File "/home/kevin/Source/muddev/evennia/evennia/commands/default/muxcommand.py", line 201, in parse
    self.character = self.caller.get_puppet(self.session)
  File "/home/kevin/Source/muddev/evennia/evennia/accounts/accounts.py", line 376, in get_puppet
    return session.puppet
AttributeError: 'NoneType' object has no attribute 'puppet'
```

#### To Reproduce
1. Generate a new world with `evennia --init mygame`
2. cd mygame
3. evennia migrate
3. Update server/config/settings.py to add `MULTISESSION_MODE = 2` and `MAX_NR_CHARACTERS = 2`
4. evennia start
5. create super user account
6. log into game as super user
7. `ic Superuser` to puppet the character
8. `batchcommand tutorial_world.build`
9. Observe the stack trace

Details from the server.log file:

```
20-11-14 18:49:27 [::] Traceback (most recent call last):
20-11-14 18:49:27 [::]   File "/home/kevin/Source/muddev/evennia/evennia/commands/cmdhandler.py", line 640, in _run_command
20-11-14 18:49:27 [::]     yield cmd.parse()
20-11-14 18:49:27 [::]   File "/home/kevin/Source/muddev/evennia/evennia/commands/default/muxcommand.py", line 201, in parse
20-11-14 18:49:27 [::]     self.character = self.caller.get_puppet(self.session)
20-11-14 18:49:27 [::]   File "/home/kevin/Source/muddev/evennia/evennia/accounts/accounts.py", line 376, in get_puppet
20-11-14 18:49:27 [::]     return session.puppet
20-11-14 18:49:27 [::] AttributeError: 'NoneType' object has no attribute 'puppet'
20-11-14 18:49:27 [EE] User input was: 'unquell'.
```

#### Expected behavior

Running the `batchcommand tutorial_world.build` script should not generate stack traces. The problem does not happen with `
MULTISESSION_MODE = 0`.

#### Environment, Evennia version, OS etc

evennia -v:
```
    Evennia 0.9.5 (rev 5f361fbd5) (rev 5f361fbd5)
    OS: posix
    Python: 3.8.5
    Twisted: 20.3.0
    Django: 2.2.17
```

OS: Ubuntu 20.04.1 LTS

**Comment by kcoram** [comment]
I have been able to track what is happening, but I don't know enough about the framework yet to know what should happen.

When leaving the tutorial OutroRoom, this code from contrib/tutorial_world/rooms.py is executed:

```
    def at_object_leave(self, character, destination):
	if character.account:
	   character.account.execute_cmd("unquell")
```
`execute_cmd()` is defined in accounts/account.py:
```
   def execute_cmd(self, raw_string, session=None, **kwargs):
     . . . .
        if not session and _MULTISESSION_MODE in (0, 1):
            # for these modes we use the first/only session                               
            sessions = self.sessions.get()
	   session = sessions[0] if sessions else None

        return cmdhandler.cmdhandler(
            self, raw_string, callertype="account", session=session, **kwargs
        )
```
Since no session is passed in from the `at_object_leave()` function, the `if not session ...` code runs. There is no block for MULTISESSION_MODE 2 (or 3), so the `session` variable remains undefined. When the "unquell" command tries to run, it attempts to get the puppet from the session, but without a session it fails with the stack trace in the OP.

I could hack around the problem by removing the `and _MULTISESSION_MODE in (0, 1)` conditional, but I doubt that is actually a good and proper solution. There must be an appropriate way to get the correct session for MULTISESSION_MODE 2, but none has yet come to mind.

**Comment by Vikka** [comment]
I have the same error in `MULTISESSION_MODE = 3`.

**Comment by kcoram** [comment]
Fixed by #2255 2255
